src/templates/organization.js

```diff
@@ -153,7 +155,10 @@ export default function OrganizationTemplate({ data }) {
         <div className="mb-10 md:mt-6 flex flex-col lg:flex-row">
           <div className="w-5/5 lg:w-3/5">
             <div className="flex">
-              <div className="mr-5 w-16 flex-shrink-0 block">
+              <div
+                className="mr-5 w-16 flex-shrink-0 block"
+                style={{ display: showImg }}
```

Comment:
We are trying to use tailwind css instead of inline css whenever we can.
Here you could remove the inline style and use `classnames` to add the class name conditionally.
This allows to use a tailwind class but also to have the condition in the same place. Here it is preferable to have the condition alongside the classes, to avoid having to search for `showImg` property. Plus `showImg` sounds like a boolean while it contains a class :)

Reply:
@loiclefloch check out my comment below. I think we can avoid messing with styles entirely.